changelogs/changelog.yaml

```diff
@@ -238,3 +238,28 @@ releases:
       name: attachment
       namespace: ''
     release_date: '2022-09-21'
+  2.1.0:
+    changes:
+      bugfixes:
+      - inventory plugin - sysparm_query attribute is taken into account.
+      - mapping - When creating custom mapping, one now can again list unknown fields
```

Comment:
Same as previous comment. Drop "now". It is implied and just makes the phrasing awkward.